docs/content_management/taxonomy/taxonomy_api.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Using the PHP API you can browse taxonomy entries, get their information and manage them.
+---
+
+# Taxonomy API
+
+To manage taxonomies, use `Ibexa\Contracts\Taxonomy\Service\TaxonomyServiceInterface`.
+
+## Getting taxonomy entries
+
+To get a single taxonomy entry, you can use `TaxonomyServiceInterface::loadEntryByIdentifier()`:
```

Comment:
Maybe we can also mention other methods to get Taxonomy Entries?
* load using ID: `loadEntryById(int $id): TaxonomyEntry`
* load using underlying Content's ID: `loadEntryByContentId(int $contentId): TaxonomyEntry`
* load root (main) entry in the taxonomy: `loadRootEntry(string $taxonomyName): TaxonomyEntry`